React/tic-tac/src/redux/store.js

```diff
@@ -0,0 +1,24 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+import { loadState, saveState } from '../localStorage';
+
+import formReducer from './form/reducer';
+import reducerAuth from './auth/reducer';
+
+const rootStore = combineReducers({
+  form: formReducer,
+  auth: reducerAuth
+});
+
+const persistState = loadState();
+
+const store = createStore(rootStore, persistState, applyMiddleware(thunk));
+
+store.subscribe(() => {
```

Comment:
Nice approach! 😸
Just remember that right now, the app will work great with this, but as the app grows bigger, writing to the Local Storage every time a piece of the state is modified may give off unwanted results.